### ISSUE TITLE
Product V2 - UI improvement

### ISSUE THREAD

**Issue by MatShir** [PrestaShop/PrestaShop#31110]
### 1. Product footer https://github.com/PrestaShop/PrestaShop/issues/31586

- [x] #31586

Acceptance test:
When the product page is displayed, the footer only contains:

- go to catalog
- Save and publish
- Cancel
- 3 dot button (containing: add new product, delete, duplicate and Preview)

The height is adjusted, so we can save some little bit space

### Combination list

- [x]  Remove the reset mode because it is redundant with the cancel mode which causes exactly the same number of clicks (already done I believe)
- [x]  Display the edition mode at first click, and not at the blur (it was done on change, not on first click no reason to turn the edition mode until something has actually been edited no?)
- [x] #31587

### Combination generation

- [ ]  Close the modal after generation

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/43613217/222147757-94a8389a-c7fd-4121-9f59-dcc7cabf4716.png">

Acceptance test:When I click on generate combination, the modal close itself at the ending of the generation

### Combination modal

- [ ]  Hide the history panel by default and displays it only after at least 2 pages edit

### 3. UI mobile #31589

- [x] #31589
 
Mockups:
![image](https://user-images.githubusercontent.com/43613217/222147608-5b2f9e28-2cd0-4a70-adac-5318b7df04e0.png)

- Apply modification of display for the header based on the mockups
- We hide the references in the header in mobile
- In mobile don't make for now the header sticky
- The actions present next to the tabs (sales, product list, ...) should be accessed in the header (see if it implies having two sets of buttons that switch, or if it should always be in the header)
- put the all the buttons in the dropdown in the footer in desktop and mobile view except for 'go to catalog', 'cancel' and 'save and published' (should be most handled for desktop mode already)
- Make the cancel button as an icon in mobile view
- make the tab sticky in the header if possible (NTH)

**Comment by MatShir** [comment]
UI mobile 
<img width="390" alt="Mobile_prd_page3" src="https://user-images.githubusercontent.com/43613217/221831138-6c7707a2-e428-4fce-ae6f-0510b7d3ec6b.png">
<img width="390" alt="Mobile_prd_page2" src="https://user-images.githubusercontent.com/43613217/221831145-a01385ae-9f18-4cfb-96f1-da1e6267b460.png">
<img width="390" alt="Mobile_prd_page1" src="https://user-images.githubusercontent.com/43613217/221831150-4a289e2b-a5a0-4328-9b92-0643df917918.png">

**Comment by MatShir** [comment]
**Specs comments:**
- We hide the references in the header in mobile
- In mobile don't make for now the header sticky
- put the all the buttons in the dropdown in the footer in desktop and mobile view except for 'go to catalog', 'cancel' and 'save and published' 
- Make the cancel button as an icon in mobile view

Nice to have: 
- make the tab sticky in the header if possible

**Comment by jolelievre** [comment]
Suggested separation into different sub issues/PRs:

### 1. Product footer
- The footer has to many interactive buttons (most buttons should be in dropdown)

### 2. Combination improvement
#### Combination list
- Remove the reset mode because it is redundant with the cancel mode which causes exactly the same number of clicks (already done I believe)
- Display the edition mode at first click, and not at the blur (it was done on change, not on first click no reason to turn the edition mode until something has actually been edited no?)
- On the edition mode, out-of-scope parameter can be hidden (need to search what else should be hidden/disabled: footer, tabs, ... To be experimented).
#### Combination generation
- Add icon add and new wordings to make the all attribute selection more obvious
- Close the modal after generation
#### Combination modal
- Hide the history panel by default and displays it only after at least 2 pages edit

### 3. UI mobile
- Apply modification of display for the header based on the mockups
- We hide the references in the header in mobile
- In mobile don't make for now the header sticky
- The actions present next to the tabs (sales, product list, ...) should be accessed in the header (see if it implies having two sets of buttons that switch, or if it should always be in the header)
- put the all the buttons in the dropdown in the footer in desktop and mobile view except for 'go to catalog', 'cancel' and 'save and published' (should be most handled for desktop mode already)
- Make the cancel button as an icon in mobile view
- make the tab sticky in the header if possible (NTH)
